lib/ansible/modules/windows/win_domain_computer.py

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2017, AMTEGA - Xunta de Galicia
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+---
+module: win_domain_computer
+short_description: Manage computers in Active Directory
+description:
+  - Create, read, update and delete computers in Active Directory using a
+    windows brigde computer to launch New-ADComputer, Get-ADComputer,
+    Set-ADComputer, Remove-ADComputer and Move-ADObject powershell commands.
+options:
+  name:
+    description:
+      - Specifies the name of the object. This parameter sets the Name property
+        of the Active Directory object. The LDAP display name (ldapDisplayName)
+        of this property is name.
+    required: true
+  sam_account_name:
+    description:
+      - Specifies the Security Account Manager (SAM) account name of the
+        computer. It maximum is 256 characters, 15 is advised for older
+        operating systems compatibility. The LDAP display name
+        (ldapDisplayName) for this property is sAMAccountName.
+        Note. All computer SAMAccountNames needs to end with a $.
+    default: C(name)
```

Comment:
I like to keep the `default` option as defaulted hard coded values. Use the description to say if omitted then the value is the same as `C(name)`.